ray-operator/test/support/utils.go

```diff
@@ -16,6 +16,8 @@ type OutputType string
 
 const (
 	Log OutputType = "log"
+	// MultiKueueController represents the vaue of the MultiKueue controller
+	MultiKueueController = "kueue.x-k8s.io/multikueue"
```

Comment:
Consider just inlining this string in the tests since it's only referenced once or twice

Reply:
ok, done